river/test_.py

```diff
@@ -36,6 +36,8 @@ def get_all_estimators():
         compose.FuncTransformer,
         compose.Pipeline,
         compose.Grouper,
+        ensemble.AdaptiveRandomForestClassifier,
+        ensemble.AdaptiveRandomForestRegressor,
```

Comment:
Should we shorten their names and follow the path paved by SRPClassifier?

Reply:
Personally, I prefer this convention as it is clear what it means.

This goes down to the mental load on the user, are these acronyms well know in the community? A counterexample is the `PAClassifier`. This is especially tricky with new methods. 

I don't oppose using `ARFClassifier` or `ARFRegressor` but is it really a big deal?

To a lesser degree, we can also account for the pronunciation of an acronym, `ADWIN`has a nice ring to it, on the other hand, `ARF` just sounds weird to me.

Reply:
Yes, it does sound weird hahaha

Well, the best solution is keeping the current names.